nvtabular/workflow/workflow.py

```diff
@@ -78,7 +78,9 @@ def __init__(self, output_node: WorkflowNode, client: Optional["distributed.Clie
         self.graph = Graph(output_node)
         self.executor = DaskExecutor(client)
 
-    def transform(self, dataset: Dataset) -> Dataset:
+    def transform(
+        self, dataset: Union[Dataset, "cudf.DataFrame", pd.DataFrame]
```

Comment:
The dataframe types are usually represented with `DataFrameType` from Core in the NVT code base. I'm not sure what we want to do about that yet—it might be something we can clean up either wit Protocols or using our new dispatch tools—but for now I'd follow the style of the existing code, just to make it easier to find all the places that need to change later.